docs/concepts/lifecycle.md

```diff
@@ -1 +1,110 @@
+
+
+The ```Package``` task's dependencies are checked next. ```Package``` depends on ```Test```, and that dependency is
+consequently pushed onto the stack.
+
+The stack can be represented as such:
+
+|Stack Position | Target            | Dependency Task |
+|:-------------:|-------------------|-----------------|
+|2              |Package            | Test            |
+|1              |Build              | Package         |
+
+
+```Test``` in turn depends on ```Compile```, which depends on ```Prepare```, which depends on ```Version```, which
+depends on ```Initialize```, which depends on ```Clean```.
+
+The resulting stack can be represented as such:
+
+|Stack Position | Target            | Dependency Task |
+|:-------------:|-------------------|-----------------|
+|8              |Clean              | N/A             |
+|7              |Initialize         | Clean           |
+|6              |Version            | Initialize      |
+|5              |Prepare            | Version         |
+|4              |Compile            | Prepare         |
+|3              |Test               | Compile         |
+|2              |Package            | Test            |
+|1              |Build              | Package         |
+
+
+Given LIFO, the stack unravels as each task is completed successfully. ```Clean``` is called, followed by
+```Initialize```, followed by ```Version```, followed by ```Prepare```, followed by ```Compile```, followed by
+```Test```, followed by ```Package```, followed by ```Build```.
+
+
+## Publish lifecycle
+The ```Publish``` lifecycle closely follows the ```Build``` lifecycle. ```Publish``` first depends on ```Verify``` and
+```Document```. ```Verify``` depends on ```Package```, and ```Document``` depends on ```Version```.
+
+The resulting stack can be represented as such:
+
+|Stack Position | Target            | Dependency Task |
+|:-------------:|-------------------|-----------------|
+|10             |Clean              | N/A             |
+|9              |Initialize         | Clean           |
+|8              |Version            | Initialize      |
+|7              |Prepare            | Version         |
+|6              |Compile            | Prepare         |
+|5              |Test               | Compile         |
+|4              |Package            | Test            |
+|3              |Verify             | Package         |
+|2              |Document           | Version         |
+|1              |Publish            | Verify, Document|
+
+The publish lifecycle stack unravels in kind. ```Clean``` is called, followed by ```Initialize```, followed by
+```Version```, followed by ```Prepare```, followed by ```Compile```, followed by ```Test```, followed by ```Package```,
+followed by ```Verify```. All of the dependencies of ```Document``` have also been resolved, so ```Document``` is also
+successfully completed. ```Publish``` is executed last to complete the publish lifecycle.
+
+
+## Copyright and License
```

Comment:
no need to put this at the bottom of the file; it'll be included in the footer of the docfx generated site

Reply:
deleting the copyright & license section